app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/index.test.tsx

```diff
@@ -0,0 +1,207 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { act, fireEvent, render, screen } from "@testing-library/react";
+import React from "react";
+import { BaseSubject, mockChangelog } from "./mocks";
+
+const Subject = ({
+  rejectChange = () => {},
+  approveChange = () => {},
+  startRemoteSettingsApproval = () => {},
+  ...props
+}: React.ComponentProps<typeof BaseSubject>) => (
+  <BaseSubject
+    {...{
+      rejectChange,
+      approveChange,
+      startRemoteSettingsApproval,
+      ...props,
+    }}
+  />
+);
+
+describe("ChangeApprovalOperations", () => {
+  it("renders as expected", async () => {
+    render(<Subject />);
+    await screen.findByTestId("action-button");
+  });
+
+  it("when user can review, supports approval and opening remote settings", async () => {
+    const approveChange = jest.fn();
+    const startRemoteSettingsApproval = jest.fn();
+    render(
+      <Subject
+        {...{
+          canReview: true,
+          reviewRequestEvent: mockChangelog(),
+          approveChange,
+          startRemoteSettingsApproval,
+        }}
+      />,
+    );
+    const approveButton = await screen.findByTestId("approve-request");
+    await act(async () => void fireEvent.click(approveButton));
```

Comment:
Shouldn't need to wrap `fireEvent` calls in `act` since it [does it already](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily). You may need to `waitFor` the below assertion, though